.github/workflows/makefile.yml

```diff
@@ -82,6 +121,12 @@ jobs:
         mpirun -np ${{matrix.np}} ../../bin/Conquest
         cat Conquest_out
 
+    # - name: Run test 004
```

Comment:
Why is this test commented out?

Reply:
Because it's broken, I'll just remove the commented out things